### ISSUE TITLE
SQL Server stored procedure create code and routine body is invalid.

### ISSUE THREAD

**Issue by elky84** [HeidiSQL/HeidiSQL#667]
## Steps to reproduce this issue

SQL Server stored procedure create code and routine body is invalid.
always.

## Current behavior

i go to procedure tab.
but i can't show all stored procedure content.
i don't know standard for stored procedure content.

![image](https://user-images.githubusercontent.com/4001940/58843571-0fdbb800-86ae-11e9-8201-355ae91661e0.png)

![image](https://user-images.githubusercontent.com/4001940/58843584-1c601080-86ae-11e9-9fa2-6d016cfcee0f.png)

## Expected behavior

show all stored procedure content.

## Possible solution

EXEC sp_helptext 'dbo.P_Guild_Present_Point_Add';

![image](https://user-images.githubusercontent.com/4001940/58843559-ff2b4200-86ad-11e9-9e68-5ee733442f3c.png)

to above text to show all procedure code.

## Environment

* **HeidiSQL version:** 
  10.1.0.5581 64bit
* **Database system and version:** 
  Microsoft SQL Server 2016 (RTM-GDR) (KB4019088) - 13.0.1742.0 (X64) 
	Jul  5 2017 23:41:17 
	Copyright (c) Microsoft Corporation
	Developer Edition (64-bit) on Windows 10 Enterprise 6.3 <X64> (Build 17763: )

* **Operating system:** 
  Windows 10 Enterprise 64bit

**Comment by rikin-patel435** [comment]
Any update on this issue?

**Comment by elky84** [comment]
good works!
thanks regard!!

Sorry for the late handling

**Comment by rikin-patel435** [comment]
Thanks... 👍 

The most awaited fix is finally done ...

**Comment by rikin-patel435** [comment]
I have installed 11.0.0.6113 but issue is not fixed. I think It just closed.

**Comment by ansgarbecker** [comment]
Not sure why it was closed - @elky84  did you re-test and it worked for you?

**Comment by elky84** [comment]
@ansgarbecker sorry. not fixed.

**Comment by NeV3rKilL** [comment]
This shoud be not difficult to fix. Can I have some info about the files where is this. I could check it and send a pull request.

**Comment by terenzif** [comment]
I would be pleased to make a donation to this great app, but in leaving this bug open you show absolutely no respect for mssql users!!!

**Comment by ansgarbecker** [comment]
@terenzif no, you're wrong - I respect every user. Only the things in MS SQL work very different, compared to my favorites MySQL and MariaDB. Feel free to post something more constructive.
